backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/backup/HybridBackupCommand.java

```diff
@@ -135,7 +135,16 @@ protected void executeCommand() {
     @Override
     public boolean performNextOperation(int completedChildCount) {
         restoreCommandState();
-        if (getParameters().getVmBackup().getPhase() == VmBackupPhase.STARTING) {
+        VmBackupPhase phase = getParameters().getVmBackup().getPhase();
+
+        if ((getParameters().getVmBackup().isStopped() && !phase.isBuckupCleaningUp())
```

Comment:
is special handling needed if the transfer is in progress?

Reply:
Good question. Is it? Arik suggested that stop during transfer handling will be done in a separate patch, so I didn't concentrate on that. But also that is why I have a feeling that I'm missing something here

Reply:
I didn't test it, but it sounds like it could cause issues because transfer uses the snapshot

Reply:
But currently with stopping by changing the backup phase externally we don't handle it anyway, right?

Reply:
I mean I don't see any handling

Reply:
it is not handled, let's see how it behaves, I believe that the transfer would be canceled because of an error which isn't too bad if that's the case but maybe we can do something more graceful

Reply:
to check it you can set a breakpoint in https://github.com/oVirt/ovirt-engine/blob/509ecc5e666f7a0954cb6139ddf905aba44daac6/backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/image/TransferDiskImageCommand.java#L739

then finalize the backup, and continue the TransferDiskImage thread after the snapshot is removed